### PR TITLE
feat: price list details

### PR DESCRIPTION
**Pull request by zakariaelas**
### What

- adds price list detail header + prices + raw price list sections

### Notes

- "Edit price list details" coming in a different PR (depends on price list form context ticket)
- "Edit Manually" functionality coming in a different PR 
- "Import price list" functionality on hold for now
- Price row "remove product" action on hold
- Price row "Edit prices" dependent on grid editor #509 
- Price row on click dependent on #445

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,8 +11,8 @@
   "dependencies": {
     "@emotion/core": "^10.0.28",
     "@emotion/styled": "^10.0.27",
-    "@medusajs/medusa": "^1.2.1",
-    "@medusajs/medusa-js": "^1.1.1",
+    "@medusajs/medusa": "1.2.1-dev-1649697145856",
+    "@medusajs/medusa-js": "1.1.1-dev-1649697145856",
```
Comment (srindom) [inline package.json:15]: todo: make sure that you clean up after having used `medusa-dev`

**Review by srindom — CHANGES_REQUESTED** [review]
I am unable to get this up and running. Getting this error - have tried to clean :

```
ModuleNotFoundError: Module not found: Error: Can't resolve '../../../selectable-table' in '[..]/src/domain/pricing/details/sections/prices/prices-table'
```

**Comment by zakariaelas** [comment]
@srindom apologies I accidentally left out a file 😅 . Btw, the `SelectableTable` comes from our work on discount conditions.

**Review by srindom — APPROVED** [review]
LGTM!

**Pull request merged**